config/drupal-8/drupal-8-all-deprecations.php

```diff
@@ -27,7 +27,7 @@
     // the following rule adds unneeded annotations.
     $rectorConfig->services()->remove(AddDoesNotPerformAssertionToNonAssertingTestRector::class);
 
-    $rectorConfig->bootstrapFiles([
+    $rectorConfig->autoloadPaths([
         __DIR__ . '/../drupal-phpunit-bootstrap-file.php'
```

Comment:
We're not passing in paths here; we need a bootstrap file that executes PHP code. The autoload paths scan directories instead. Was there a reason?